CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 * ### ALL
     * #### Added
+        * Add `session.lock_session()` and `session.unlock_session()` to all drivers except nimodinst - [#846](https://github.com/ni/nimi-python/issues/846)
```

Comment:
If we grab session lock, then it should be called out here?

> * Fix thread-safety issues by using IVI session lock

or something like that.

Reply:
Added